src/test/java/games/strategy/triplea/delegate/WW2V3_42_Test.java

```diff
@@ -94,6 +96,7 @@ public void testBombAndAttackEmptyTerritory() {
 
   @Test
   public void testLingeringSeaUnitsJoinBattle() throws Exception {
+    ProData.initializeSimulation(new ProAI("one","two"), m_data, germans(m_data));
```

Comment:
@simon33-2 Maybe I was unclear. I meant we need to take into account not just the defense value but also support and that the AI code was an example of how to calculate a group of units total defense value (rather than just checking each individual units defense attribute). I definitely didn't intend for the AI code to be referenced from outside of the AI. That is really going down a dangerous path of coupling AI code into engine logic.